### PR TITLE
docs(PPDSC-2690): update drawer storybook

### PR DESCRIPTION
**Pull request by mutebg**
PPDSC-2690

**What**

1. Background - why this is needed
2. What did you do
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by pp-serviceaccount** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2690-update-drawer-stories
  Storybook: http://ncu-newskit-docs-pr.s3-website-eu-west-1.amazonaws.com/ppdsc-2690-update-drawer-stories/storybook

**Review by Vanals — DISMISSED** [review]

**Review by JohnTParsons — APPROVED** [review]

File: src/drawer/__tests__/drawer.stories.tsx [hunk 1]
```diff
@@ -108,350 +32,299 @@ const useActiveState = (
...
-    const [isActive, setIsActive] = React.useState(false);
-    const [placement, setPlacement] = React.useState('left');
 
-    const open = () => setIsActive(true);
-    const close = () => setIsActive(false);
+type Placement = NonNullable<DrawerProps['placement']>;
 
-    const onChangeValue = (ev: React.ChangeEvent<HTMLDivElement>) =>
-      setPlacement((ev.target as HTMLInputElement).value);
+const DRAWER_HEDER = 'Drawer Title';
```
Comment (JohnTParsons) [inline src/drawer/__tests__/drawer.stories.tsx:38]: Spelling: DRAWER_HEADER

**Review by JohnTParsons — CHANGES_REQUESTED** [review]
Spelling of DRAWER_HEADER

File: src/drawer/__tests__/drawer.stories.tsx (hunk 1 above)
Comment (mutebg) [reply src/drawer/__tests__/drawer.stories.tsx]: thanks updated

**Review by JohnTParsons — APPROVED** [review]

**Review by Vanals — APPROVED** [review]

**Pull request merged**